x-pack/auditbeat/module/system/user/user_test.go

```diff
@@ -0,0 +1,34 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package user
+
+import (
+	"testing"
+
+	"github.com/elastic/beats/auditbeat/core"
+	mbtest "github.com/elastic/beats/metricbeat/mb/testing"
+)
+
+func TestData(t *testing.T) {
```

Comment:
Should a function to generate test data be shipped as part of Auditbeat?

Reply:
I'm not sure what you mean - this function does generate test data. Do you mean we shouldn't (most other metricsets have this) or that it should be different?

Reply:
Oh jeez, I missed the _test part of the file name 🤦‍♂️  Never mind, I'm not used to the Go conventions of having test files right besides their code file.